mico-core/src/main/java/io/github/ust/mico/core/model/MicoServiceDeploymentInfo.java

```diff
@@ -73,8 +73,9 @@
 
     /**
      * The instance id of this deployment.
-     * It is used to be able to have multiple independent deployments
-     * of the same MICO service.
+     * It will be generated immediately after the creation of this {@code MicoServiceDeploymentInfo}.
```

Comment:
This must be instead of it will be because the current comment implies that the class itself generates this!